docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+ * **Simplified ordering node creation process**: Because a genesis block no longer needs to exist before an ordering node is created, admins can now focus on the process of setting up their infrastructure and making sure their node is working correctly before joining a particular application channel. Additionally, because an ordering node no longer needs to scan the system channel ledger to determine which channels it is joined to and then replicate those ledgers, the node starts up more rapidly.
+
+**Benefits of the new process:**
+
+ * **Increased privacy**: Because all ordering nodes used to be joined to the system channel, every ordering node in a network knew about the existence of every channel on that ordering service, even if the node itself wasn't a consenter on that channel and therefore wasn't ordering or storing its blocks. Now, an ordering node only knows about the channels it is joined to.
+ * **Scalability**: When there is a large number of ordering nodes on the system channel, starting the node can take a long time as it has to wait for each orderer to replicate the system channel ledger showing it has joined. Additionally, large numbers of nodes on the system channel increases the number of "alive" messages being sent between nodes. In large system channels with a large number of nodes, this can cause processing issues, even if no application channels have a large number of ordering nodes on it.
+ * **Flexibility**: Whereas ordering nodes could only be part of a single ordering service, orderers can now join or leave channels as needed, similar to how peers can join any channel its organization is a member of.
+ * **Operational benefits**:
+    * Easy to list the channels that the ordering node is a consenter on.
+    * Simple process to delete a channel and the ledger associated with that channel.
+    * Ordering nodes now detect that they have been added to a channel more quickly. Previously, this process could take several minutes, causing ordering node admins to restart their nodes so that the node could detect that it was joined more quickly.
+    * If the MSP of a peer organization, as listed in the system channel, needs to be changed, the peer organization no longer needs to coordinate with an orderer organization to change the MSP.
+
+**Note: the new mode of creating channels is incompatible with creating a channel using a system channel. If the system channel exists, the `Join` and `Remove` operations will not be supported. Similarly, they cannot be used with a Solo or Kafka ordering service.**
```

Comment:
Maybe it's too detailed here, but `Remove` works, but only for the system channel.